x-pack/plugins/ingest_manager/common/services/datasource_to_agent_datasource.test.ts

```diff
@@ -140,6 +140,44 @@ describe('Ingest Manager - storedDatasourceToAgentDatasource', () => {
     });
   });
 
+  it('returns agent datasource config without disabled streams', () => {
```

Comment:
Maybe we can add the test with disabled stream?

Reply:
what do you mean? this test is a case where a stream is disabled (the disabled stream does not show up in final agent config)